archway/chain.json

```diff
@@ -37,33 +37,33 @@
   },
   "codebase": {
     "git_repo": "https://github.com/archway-network/archway",
-    "recommended_version": "v8.0.0",
+    "recommended_version": "v9.0.0",
```

Comment:
Although the automated verification is yet to come, we'll be requiring that versions referenced here be defined in the versions.json file. Please add v8 and v9 there, as well.